src/components/text/Content.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components/native'
```

Comment:
You are right about the naming i just thought we could get some wrong import if we name it equal to the react-native `Text`. But i can adjust that

According to label-parameter i think maybe yes it can be a little bit more intuitive but we really get long names. For web f.e. there are also <h1> tags and <p> and so on. Maybe we should discuss that in a small call together.

Reply:
Should be clear, because of different import, but i wouldn't mind if it is "DefaultText" or so.

I see us running in something like SubSubHeading or stuff like this... hmh

Reply:
okay i'm fine with `Text` then to keep it as short as possible

Reply:
I keep for now `content` until we find a better solution. then we should also rename `content` in the `fonts.ts`